pkg/runtime/runtime.go

```diff
@@ -1942,53 +2039,68 @@ func (a *DaprRuntime) stopActor() {
 	}
 }
 
+func (a *DaprRuntime) shutdownComponentInstance(name string, instance interface{}, category ComponentCategory) error {
+	switch category {
+	case bindingsComponent:
+		if outputBinding, ok := instance.(bindings.OutputBinding); ok {
+			if err := a.shutdownOutputBinding(name, outputBinding); err != nil {
+				return err
+			}
+		}
+		if inputBinding, ok := instance.(bindings.InputBinding); ok {
+			if err := a.shutdownInputBinding(name, inputBinding); err != nil {
```

Comment:
```suggestion
			return  a.shutdownInputBinding(name, inputBinding)
```

Reply:
We don't want to return when it's `nil`.

Reply:
Sorry, I did not get it earlier, fixed!